js/scenes/Profile/ProfileContainer.js

```diff
@@ -1,13 +1,117 @@
 import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { ImagePickerIOS } from 'react-native';
 import Profile from './Profile';
+import { 
+  updateBioField,
+  updateGoalsField,
+  updateSocialMedia,
+  updateImageUrlField
+} from '../../redux/modules/user-forms';
+
+import { customFieldsUpdater } from '../../config/helpers';
+import { auth } from '../../config/firebase';
 
 class ProfileContainer extends Component {
 
+  static route = {
+    navigationBar: {
```

Comment:
If the component doesn't hold `state` or life cycle methods, opt for function components